tox.ini

```diff
@@ -1,15 +1,18 @@
 [tox]
-envlist = py27
-
-[testenv:py27]
-deps =
-    mock
-    mozfile >= 1.1
-    mozprocess >= 0.22
-    pylama
-    pytest
-    wptserve >= 1.4.0
+envlist = py27, coverage
 
+[testenv]
+deps = -rrequirements/tests.txt
 commands =
     pylama mozdownload
-    py.test {posargs}
```

Comment:
This is a really weird one. When I keep that line on my machine, it runs the test twice, which is just not efficient.

But it looks that on Travis it then omits them entirely... I will have to add it and see the output from Travis then...